src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write your code here
+const headers = Array.from(document.querySelector('tr').children);
+
+const parentHeader = headers[0].parentElement;
+const clonedHeader = headers[1].cloneNode(true);
+
+parentHeader.insertBefore(clonedHeader, headers[4]);
```

Comment:
Ensure that the `headers` array has at least 5 elements before attempting to access `headers[4]`. If the array is shorter, this will cause an error.